Error mongodb-client-encryption at bundle script

## I'm submitting a...

<!-- Put an `x` in all the boxes that apply -->

- [ ] Bug report
- [ ] Feature request
- [ ] Documentation issue or request
- [X] Other... Please describe:

## Current Behavior

Error thrown at `build` script:

![image](https://user-images.githubusercontent.com/52841892/180431589-5ef34009-c060-48d5-a443-096b22fa37f6.png)

I don't know much about it. Should we add `mongodb-client-encryption` in MongoDb integration?

I don't know the implications of `--external:mongodb-client-encryption`

### Error Log

```
> esbuild --bundle --outfile=bundle/index.js --sourcemap --minify --keep-names --platform=node --target=node14 --format=cjs  --external:aws-sdk --external:@oclif/* --external:@jovotech/cli* --external:@alexa/* "src/app.desarrollo.ts"
✘ [ERROR] Could not resolve "mongodb-client-encryption"
    node_modules/.pnpm/mongodb@4.8.0/node_modules/mongodb/lib/utils.js:1126:42:
      1126 │ ... mongodbClientEncryption = require('mongodb-client-encryption');
           ╵                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~
  You can mark the path "mongodb-client-encryption" as external to exclude it from the bundle, which will remove this error. You can also surround this "require" call with a try/catch block to handle this failure at run-time instead of bundle-time.
1 error
npm ERR! code ELIFECYCLE```

## Your Environment

<!--- Include as many relevant details about the environment where the bug was discovered -->

- Jovo Framework version used: 4.2.20
- Operating System: Win10, Gitlab pipeline, everywhere

--- a/integrations/db-mongodb/docs/README.md
+++ b/integrations/db-mongodb/docs/README.md
@@ -115,3 +115,18 @@ async someHandler() {
     const newDb = client.db("anotherDb");
 }
 ```
+
+
+## Deployment
+
+If you run into a problem during [deployment](https://www.jovo.tech/docs/deployment), you may be related to the `mongodb-client-encryption` dependency of MongoDb (see [this issue](https://github.com/jovotech/jovo-framework/issues/1381) for more information).
+
+To solve this, add `--external:mongodb-client-encryption` to the [`bundle` script](https://www.jovo.tech/docs/deployment#bundle) of your Jovo app's `package.json` (find an [example here](https://github.com/jovotech/jovo-v4-template/blob/master/package.json)).
+
+```json
+{
+  "scripts": {
+    "bundle": "[...] --external:mongodb-client-encryption",
+  }
+}
+```

